editor.planx.uk/src/pages/FlowEditor/components/Settings/ServiceSettings/FlowDescription/FlowDescription.tsx

```diff
@@ -0,0 +1,78 @@
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import { useFormik } from "formik";
+import { useToast } from "hooks/useToast";
+import React from "react";
+import InputLabel from "ui/editor/InputLabel";
+import SettingsSection from "ui/editor/SettingsSection";
+import Input from "ui/shared/Input/Input";
+
+import { useStore } from "../../../../lib/store";
+import { SettingsForm } from "../../shared/SettingsForm";
+
+const FlowDescription = () => {
+  const [flowDescription, updateFlowDescription] = useStore((state) => [
+    state.flowDescription,
+    state.updateFlowDescription,
+  ]);
+  const toast = useToast();
+
+  const formik = useFormik<{ description: string }>({
+    initialValues: {
+      description: flowDescription || "",
+    },
+    onSubmit: async (values, { resetForm }) => {
+      const isSuccess = await updateFlowDescription(values.description);
+      if (isSuccess) {
+        toast.success("Description updated successfully");
+        resetForm({ values });
+      }
+      if (!isSuccess) {
+        formik.setFieldError(
+          "description",
+          "We are unable to update the service description, check your internet connection and try again",
+        );
+      }
+    },
+  });
+
+  return (
+    <Box mb={2}>
+      <SettingsSection>
+        <Typography variant="h2" component="h3" gutterBottom>
+          Service Information
+        </Typography>
+        <Typography variant="body1">
+          Useful information about this service.
+        </Typography>
+      </SettingsSection>
+      <SettingsForm
+        legend="Service Description"
+        formik={formik}
+        description={
+          <>
+            A short blurb on what this service is, how it should be used, and if
+            there are any dependencies related to this service.
+          </>
+        }
+        input={
+          <>
+            <InputLabel label="Description" htmlFor="description">
+              <Input
```

Comment:
Following the pattern we've established in https://github.com/theopensystemslab/planx-new/pull/3933 we should make this a larger text input if as we're expecting more than a few words here (we are!).

Two other question (maybe already discussed / agree and I've missed them) -
 - should this be just plain text, not rich text?
 - should there be a limit on the length of the input?

Reply:
Great shout, added the multiline prop now

For the questions, my initial thought was this was just laying the groundwork and these types of answers would come once we have a better idea when / how we display it to the user, so for now I just kept it to it's simplest implementation. Do you think we need to take it further in this PR?

Reply:
I think this is fine for now - just checking it wasn't a decision somewhere I'd missed. This will be easy to improve in future as you say if required.